openmetadata-spec/src/main/resources/json/schema/configuration/airflowConfiguration.json

```diff
@@ -23,7 +23,7 @@
       "type": "string"
     },
     "metadataApiEndpoint": {
-      "description": "Metadata api endpoint",
+      "description": "Metadata api endpoint. To be deprecated.",
```

Comment:
Is it expected to be deprecated but not the one in `argoConfiguration.json`?

Reply:
you're right, killing that from Argo